obs2ioda-v2/src/cxx/netcdf_attribute.cc

```diff
@@ -45,6 +45,14 @@ namespace Obs2Ioda {
         }
     }
 
+    int netcdfPutAttIntArray(
+        int netcdfID, const char *attName, const int *attValue, const int attLen, const char *varName, const char *groupName
```

Comment:
It might be nice to wrap the argument list in a way that's consistent with what is done for other functions, e.g., `netcdfPutAttInt`, in this file.